sql-plugin/src/main/scala/org/apache/spark/sql/rapids/RapidsShuffleInternalManagerBase.scala

```diff
@@ -601,12 +599,12 @@ abstract class RapidsShuffleThreadedReaderBase[K, C](
   private val shuffleReadTimeNs = sqlMetrics.get(METRIC_SHUFFLE_READ_TIME)
   private val dataReadSize = sqlMetrics.get(METRIC_DATA_READ_SIZE)
 
-  private var shuffleReadRange: NvtxRange =
-    new NvtxRange("ThreadedReader.read", NvtxColor.PURPLE)
+  private var shuffleReadRange: NvtxId = NvtxRegistry.THREADED_READER_READ
```

Comment:
nit, why not have `push()` return `this`, so this line becomes:
```
private var shuffleReadRange: NvtxId = NvtxRegistry.THREADED_READER_READ.push()
```
and we don't need the separate call.